internal/services/springcloud/spring_cloud_accelerator_resource.go

```diff
@@ -27,9 +28,14 @@ type SpringCloudAcceleratorModel struct {
 
 type SpringCloudAcceleratorResource struct{}
 
+func (s SpringCloudAcceleratorResource) DeprecationMessage() string {
+	return features.DeprecatedInFivePointOh("Azure Spring Apps is now Deprecated and will be Retired on 2028-05-31 - as such the `azurerm_spring_cloud_accelerator` resource is deprecated and will be removed in v6.0 of the AzureRM Provider. See https://aka.ms/asaretirement for more information.")
```

Comment:
Without visibility into usage we feel that removing a deprecated resource ~3 years before its retirement date is too early. Can we update this to say it will be removed in a future major version. We can reassess closer to when 6.0 is slated for release and update the deprecation message accordingly then.
```suggestion
	return features.DeprecatedInFivePointOh("Azure Spring Apps is now deprecated and will be retired on 2028-05-31 - as such the `azurerm_spring_cloud_accelerator` resource is deprecated and will be removed in a future major version of the AzureRM Provider. See https://aka.ms/asaretirement for more information.")
```